sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -394,6 +394,18 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ADAPTIVE_EXECUTION_LOG_LEVEL = buildConf("spark.sql.adaptive.logLevel")
+    .internal()
+    .doc("Configures the log level for adaptive execution logging. The value can be 'trace', " +
```

Comment:
nit: `for adaptive execution logging of plan changes.` as it doesn't affect the logging in adaptive execution optimization rules.